WPMedia/fusion-news-theme-blocks
# Pen 802 small promo styling

## Description
- update to set small promo to 100% width in various breakpoints
- set paddings

## Jira Ticket
- [PEN-802](https://arcpublishing.atlassian.net/browse/PEN-802)

## Acceptance Criteria
- see comments from https://arcpublishing.atlassian.net/browse/PEN-802?focusedCommentId=475860

## Test Steps
- create new page and add small promo or small manual promo
- test on image left/ right/ top/ bottom to have md-padding (1.5 rem)
- the width of the image should be 100% width 

## Effect Of Changes
### Before
_Example: When I clicked the search button, the button turned red._

[include screenshot or gif or link to video, storybook would be awesome]

### After
<img width="848" alt="Screen Shot 2020-11-13 at 9 02 13 PM" src="https://user-images.githubusercontent.com/4411687/99142019-e09cda80-2605-11eb-8bf2-d1d8ddc15d89.png">
<img width="504" alt="Screen Shot 2020-11-13 at 9 03 33 PM" src="https://user-images.githubusercontent.com/4411687/99142021-e2669e00-2605-11eb-886c-a6b4b06ba361.png">


## Dependencies or Side Effects
_Examples of dependencies or side effects are:_
- Additional settings that will be required in the blocks.json
- Changes to the custom fields which will require users to reconfigure features
- Update to css framework or SDK
- Dependency on another PR that needs to be merged first

## Review Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).
